core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -358,10 +357,8 @@ public static boolean isHaMode(AlluxioConfiguration conf) {
 
     List<ConfigProperty> configs = new ArrayList<>();
     List<PropertyKey> selectedKeys =
-        conf.keySet().stream()
-            .filter(key -> GrpcUtils.contains(key.getScope(), scope))
-            .filter(key -> key.isValid(key.getName()))
-            .collect(toList());
+        conf.keySet().stream().filter(key -> GrpcUtils.contains(key.getScope(), scope))
+            .filter(key -> key.isValid(key.getName())).collect(toList());
```

Comment:
Not sure if you updated all this by hand or automatically formatted, but I think the old version of this is easier to read

Reply:
It was auto-formatted. I reverted back.